concent_api/core/models.py

```diff
@@ -404,9 +404,40 @@ class RecipientType(ChoiceEnum):
         Requestor   = 'requestor'
 
     payment_ts              = DateTimeField()
-    task_owner_key          = Base64Field(max_length = 64)
-    provider_eth_account    = Base64Field(max_length = 64)
-    amount_paid             = DecimalField(max_digits = 10, decimal_places = 2)
+    task_owner_key          = CharField(max_length = 42)
```

Comment:
Please add a constant for it and use it also in validation (and in other places if there are any).